common/test/acceptance/pages/studio/overview.py

```diff
@@ -558,6 +558,17 @@ def change_problem_release_date_in_studio(self):
         self.q(css=".action-save").first.click()
         self.wait_for_ajax()
 
+    def change_problem_due_date_in_studio(self, date):
+        """
+        Sets a new start date.
+
+        Expects date to be a string that will be accepted by the input (for example, '01/01/1970')
+        """
+        self.q(css=".subsection-header-actions .configure-button").first.click()
+        self.q(css="#due_date").fill(date)
```

Comment:
I tried this, it led to test failures. Going back to the approach used by `change_problem_release_date` above.